package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "vega-parser",
-  "version": "2.5.0",
+  "name": "@elastic/vega-parser",
+  "version": "2.5.0-kibana",
```

Comment:
Is this fork based on vega-parser 2.5? It seems like both vega-lib 3.3.1 and kibana 6.8 are using vega-parser 2.7 (it says `^2.5`, but it resolves to 2.7 if you check the yarn lock file).